README.md

```diff
@@ -96,7 +96,7 @@ Before making a pull request, please file an issue. So, other developers have th
 
 We have the following branches:
  * **gh-pages**
-   This contains shipped code. After significant features/bugfixes are accumulated on development, we make a version update and make a release.
+   This contains the autogenerated build artifacts of the master branch that is generated by Travis.This branch is used to publish susper.com .
```

Comment:
```to publish to susper.com```. And also add a space after fullstop (after Travis). You can make it more clear as: ```This branch contains the auto-generated build artifacts of master branch, generated by Travis which is used to publish to Susper.com``` Please see the ```auto-generated``` and ```Susper.com```.

Reply:
There is no need of [space] after Travis. Also for site name first letter in lower case is more suitable than capital. I have made some changes please see.